flaskr/templates/movies.html

```diff
@@ -1,2 +1,12 @@
 <title>Movie Collection</title>
-<h1>{{ movies[0] }}!</h1>
\ No newline at end of file
+<h1>Movies</h1>
+<ul>
+{% for movie in movies %}
+    <p>
+        <b>{{ movie['title'] }}</b>: 
```

Comment:
Maybe it would be better to use a `h2` because it has semantic meaning so it's better for blind user's

Reply:
Thank you.